### PR TITLE
fix: development server not detecting local changes

### PR DESCRIPTION
**Pull request by ductaily**
**Description**

Changes proposed in this pull request:

- development server stopped detecting local changes
- address suggested fixes:
  - https://github.com/asyncapi/asyncapi-react/pull/994#issuecomment-2115092868
  - https://github.com/asyncapi/asyncapi-react/pull/994#issuecomment-2115411922

**Related issue(s)**

See also <https://github.com/asyncapi/asyncapi-react/pull/994>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,8 +38,6 @@
     "bump:playground:version": "cd playground && npm --no-git-tag-version --allow-same-version version $VERSION",
     "bump:webcomp:version": "cd web-component && npm --no-git-tag-version --allow-same-version version $VERSION",
     "bump:lib:version": "cd library && npm --no-git-tag-version --allow-same-version version $VERSION",
-    "bump:version": "npm run bump:lib:version && npm run bump:webcomp:version && npm run bump:playground:version && npm run install:reactcomp:webcomponent && npm run install:reactcomp:playground",
```
Comment (derberg) [inline package.json:41]: not the whole script, just `npm run install:reactcomp:playground` please, and `npm run bump:playground:version` as we still need for others

Comment (derberg) [reply package.json:41]: I mean we still need it but as 

```
"bump:version": "npm run bump:lib:version && npm run bump:webcomp:version && npm run install:reactcomp:webcomponent",
```

File: package.json [hunk 2]
```diff
@@ -38,8 +38,6 @@
     "bump:playground:version": "cd playground && npm --no-git-tag-version --allow-same-version version $VERSION",
```
Comment (derberg) [inline package.json:38]: you can even remove this and also https://github.com/asyncapi/asyncapi-react/blob/master/playground/package.json#L3 as we do not really need to version playground, as it is just dev env, no plans to push to npm

Comment (ductaily) [reply package.json]: Sure. I removed them.

**Review by derberg — APPROVED** [review]

**Comment by pavelkornev** [comment]
@derberg can it be merged?

**Pull request merged**